{
  try {
    doExecute(refactoringContext,continuation);
  }
 catch (  Throwable t) {
    LOG.error(t);
  }
}
